examples/computer_vision/cifar10_pytorch/model_def.py

```diff
@@ -44,8 +44,8 @@ class CIFARTrial(PyTorchTrial):
     def __init__(self, context: PyTorchTrialContext) -> None:
         self.context = context
 
-        self.download_directory = "data"
-        os.makedirs(self.download_directory, exist_ok=True)
+        # self.download_directory = "data"
+        # os.makedirs(self.download_directory, exist_ok=True)
```

Comment:
what's going on with this example?

(it looks like a decent code improvement, but maybe not suitable for this PR?)

Reply:
Maybe? I thought it would make sense to show an example where we are sharing code between the training example itself and the inference example, and this allows me to share that function properly. I would think that would be the better direction to push users towards rather than just copying the `download_dataset` function wholesale into the inference example.

Reply:
I got rid of these comments

Reply:
got it, makes sense